contentreview-impl/impl/src/java/org/sakaiproject/contentreview/impl/advisors/SiteToolAdvisor.java

```diff
@@ -26,6 +26,13 @@ public boolean siteCanUseReviewService(Site site) {
         return false;
     }
 
+	public boolean siteCanUseLTIReviewService(Site site) {
+        for(String toolId : contentReviewTools)
```

Comment:
It's good practice to always bracket `if() statement` so that when someone else comes along later then don't accidentally introduce a bug by indenting some new code and not realise it's not already a block. It's just good defensive programming.